packages/headless/src/features/commerce/facets/facet-set/facet-set-reducers.ts

```diff
@@ -124,7 +124,6 @@ function restoreFacet(facetId: string) {
   return {
     facetId,
     field: facetId,
-    numberOfValues: 10,
     isFieldExpanded: false,
     preventAutoSelect: false,
     initialNumberOfValues: 10,
```

Comment:
We don't know the initialNumberOfValues either, no?...

Reply:
Yup, and we sent it to CAPI even though it's completely ignored by CAPI. The issue is tracked through https://coveord.atlassian.net/browse/CAPI-1494, we'll remove it then.